File: DSA 1/Palindrome Checker.java
import java.util.*;

// https://course.acciojob.com/idle?question=c1448792-5b79-4cce-a2e0-5ff6d39c9844

public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        //Write your code here and print the required output
	    int nc = n;
		int sum = 0;
		int no = 0;
		while(nc>0){
			no = nc%10;
			sum = sum*10 + no;
			nc/=10;
		}
		if(n == sum){
			System.out.println("true");
		}else{
			System.out.println("false");
		}
	}
}

File: Javascript/Palindrome Checker.js
// https://course.acciojob.com/idle?question=340d0f67-3062-487f-ac95-adcb744c410c

const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

readline.question('', n => {
  //Write your code here and print the required output
	// reverse the number 
	let copy = n;
	let reverse = 0;

	while(copy > 0){
		reverse = (reverse*10) + copy%10;
		copy = Math.floor(copy/10);
	}

	if(reverse == n)
		console.log(true); // same if reversed
	else
		console.log(false);
  readline.close();
});
